cmd/ocsp-updater/main.go

```diff
@@ -237,11 +240,17 @@ func (updater *OCSPUpdater) sendPurge(der []byte) {
 			ocspServer += "/"
 		}
 		// Generate GET url
+		encReq := base64.StdEncoding.EncodeToString(req)
 		urls = append(
 			urls,
-			fmt.Sprintf("%s%s", ocspServer, url.QueryEscape(base64.StdEncoding.EncodeToString(req))),
+			// OCSP generally doesn't use proper URL encoding but as far
```

Comment:
Can you do some testing so we can remove this "as far as I know?"

Can you quote chapter and verse on which version is correct? Which version do we see more of in the logs?